code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -309,6 +309,38 @@
 	afterattack(obj/target, mob/user , flag)
 		return
 
+	get_desc(mob/user)
+		if(!usr.traitHolder?.hasTrait("training_chef"))
+			return
+
+		if(src.quality >= 5)
+			. += "<br><span class='notice'>This is of great quality! The gained buffs will last longer! </span>"
+
+		if(length(food_effects) > 0)
+			. += "<br><span class='notice'> This food has the following effects: "
+			for(var/id as anything in src.food_effects)
```

Comment:
I think `as anything` is redundant here since its purpose is to skip the built in for loop typecheck but id has no type so there's no check to begin with.